contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-elixir.mdx

```diff
@@ -1,42 +1,38 @@
-Elixir supports a subset of what our [feature flag API](/docs/api/flags) provides.
+`PostHog.FeatureFlags.check/2` is the main function for checking a feature flag in Elixir.
+More documentation can be found in the [HexPM Docs](https://hexdocs.pm/posthog/PostHog.FeatureFlags.html).
```

Comment:
```suggestion
`PostHog.FeatureFlags.check/2` is the main function for checking a feature flag in Elixir. More documentation on it can be found in the [HexPM Docs](https://hexdocs.pm/posthog/PostHog.FeatureFlags.html).
```